Makefile

```diff
@@ -73,8 +73,7 @@ $(LINT_BIN):
 	@$(call print, "Fetching linter")
 	# Switch to tmp directory to prevent lnd go.mod from being modified.
 	# Hopefully someday a flag for go get will be added for this.
-	cd /tmp
-	GO111MODULE=on go get $(LINT_PKG)
+	cd /tmp && GO111MODULE=on go get $(LINT_PKG)
```

Comment:
Perhaps use `pushd` and `popd` instead so that we don't stay within `/tmp`?

Reply:
should probably make a command for this since we could use it in a number of places

Reply:
Added a command to do this for all deps. Note that each makefile line is done in a new shell, so no need to change back.